### PR TITLE
[Merged by Bors] - feat(lint-style): lint on the plain string 'Adaptation note:'

### PR DESCRIPTION
**Pull request by grunweg**
ask to use #adaptation_note instead.
This command is a unified place to annotate all commands, and avoids typos.

--------

- [x] depends on: #13665 for rewriting the existing 63 occurrences

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: scripts/lint-style.py [hunk 1]
```diff
@@ -350,6 +351,13 @@ def left_arrow_check(lines, path):
         newlines.append((line_nr, new_line))
     return errors, newlines
 
+def adaptation_note_check(lines, path):
+    errors = []
+    for line_nr, line in lines:
+        if "Adaptation note:" in line:
```
Comment (jcommelin) [inline scripts/lint-style.py:357]: Is this check safer? Because of caps and/or missing colons?
```suggestion
        if "daptation note" in line:
```

**Comment by jcommelin** [comment]
bors d+

File: scripts/lint-style.py (hunk 1 above)
Comment (grunweg) [reply scripts/lint-style.py]: Good idea! Did this and added a comment about this.

**Pull request closed without merging**